deploy/eck/charts/eck-crds/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v2
+name: eck-crds
+description: |-
+  A Helm chart for installing the ECK Custom Resource Definitions (CRD). 
+
+type: application
+
+version: 1.0.0-beta1
```

Comment:
nit: is there any reason to version these separately? I'd imagine we would want chart releases to be in step with ECK releases so might as well make version == appVersion

Reply:
No strong reason other than the need to indicate that the Helm chart is still beta quality.